File: app/security/SecurityRole.java
package security;

import be.objectify.deadbolt.core.models.Role;

/**
 * @author f.patin
 */
public class SecurityRole implements Role {
    public String name;

    @Override
    public String getName() {
        return name;
    }

    public SecurityRole(String name) {
        this.name = name;
    }
}

File: app/security/SecurityRole.scala
package security

import be.objectify.deadbolt.core.models.Role

/**
 * @author f.patin
 */
case class SecurityRole(code: String) extends Role {
	override def getName: String = code
}

object BatchRole extends Role {
  def getName: String = SecurityRole.batch
}

object SecurityRole {
	val administrator = "admin"
	val production = "production"
	val employee = "employee"
  val batch = "batch"
}
